dockerfiles/galaxy-kickstart-metavisitor/Dockerfile

```diff
@@ -20,5 +20,5 @@ CMD printenv >> /etc/default/supervisor && bash -c "source /etc/default/supervis
            --extra-vars nginx_galaxy_location=$NGINX_GALAXY_LOCATION \
```

Comment:
I tested a new image built with this Dockerfile.
The problem now is that -e $NGINX_GALAXY_LOCATION has to be specified otherwise galaxy.ini receives by default "/metavisitor" whereas nginx.conf receives nothing.
Plus... if you try to override by passing -e $NGINX_GALAXY_LOCATION="/" (or /myotherdirectory), it still fails.